Doc/library/pdb.rst

```diff
@@ -192,7 +192,7 @@ The ``run*`` functions and :func:`set_trace` are aliases for instantiating the
 access further features, you have to do this yourself:
 
 .. class:: Pdb(completekey='tab', stdin=None, stdout=None, skip=None, \
-               nosigint=False, readrc=True)
+               nosigint=False, readrc=True, invoke_origin=PdbInvokeOrigin.Unknown)
```

Comment:
```suggestion
               nosigint=False, readrc=True,
               invoke_origin=PdbInvokeOrigin.Unknown)
```

Reply:
Actually, maybe we should call these "DebuggerModes", analogous to the compilation modes (exec, eval, single, which are also related to how the program was run)?

Reply:
Well if we are doing this similar to `compile`, should we just pass in a string which is probably more familiar with Python users? We can add an argument `mode` for `pdb.Pdb` and allow `None`, `'inline'` and `'comandline'`, with `None` as the default.

A real `Enum` might be a bit overkill here.

Reply:
An enum makes typos get detected an earlier. Particularly by linters, but even by the runtime to an extent.

Reply:
I definitely agree, but it's also making it a bit harder to use. You need to do `mode = pdb.DebuggerMode.Inline`, which is not horrible. Do you think the benefit is worth the cost? I mean `compile` takes a string. Using a string as an enum is not rare in Python, but maybe we tend to use `Enum` in modern Python now?

Reply:
Good points.

I don't know whether compile works as it is simple because it's old (maybe predates enums?) or whether there was another problem.

Maybe you could ask on python-dev whether people have an opinion on this?